easyconfigs/s/SuiteSparse/SuiteSparse-5.13.0-intel-2022a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,39 @@
+name = 'SuiteSparse'
+version = '5.13.0'
+local_metis_ver = '5.1.0'
+versionsuffix = '-METIS-%s' % local_metis_ver
+
+homepage = 'https://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+toolchainopts = {'unroll': True, 'pic': True, 'oneapi_c_cxx': True}
+
+source_urls = ['https://github.com/DrTimothyAldenDavis/SuiteSparse/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['59c6ca2959623f0c69226cf9afb9a018d12a37fab3a8869db5f6d7f83b6b147d']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('M4', '1.4.19'),
+]
+
+dependencies = [
+    ('METIS', local_metis_ver),
+    ('MPFR', '4.1.0'),
+]
+
+# make sure that bin/demo can find libsuitesparseconfig.so.5 during build
+prebuildopts = "export LD_LIBRARY_PATH=%(builddir)s/SuiteSparse-%(version)s/lib:$LD_LIBRARY_PATH && "
+# remove broken symlink
+prebuildopts += "rm GraphBLAS/CUDA/test/graphblascuda_test && "
+# disable optimisation report to reduce memory use during compile
+prebuildopts += 'sed -i "s/-qopt-report=5/-qopt-report=0/" GraphBLAS/CMakeLists.txt && '
+
+# The default way to do a parallel make won't work with the Makefiles for
+# SuiteSparse, JOBS has to be set to the desired parallelism
+parallel = 1
+prebuildopts += "export JOBS=%(parallel)s && "
```

Comment:
I imagine that this won't work as intended (i.e. getting the parallel value from the job) as the true value will be overwritten by the assignment on the previous line. Need to investigate an alternate dynamic method or hardcode with a comment.

Reply:
As per https://github.com/easybuilders/easybuild-easyconfigs/commit/5f2dd890807b9b70758c70260998a2a2ae4c5912 I will hard code the `JOBS` value:

```suggestion
parallel = 1
# Adjust the JOBS value to match the available resources
prebuildopts += "export JOBS=8 && "
```